launcher/tasks/Task.h

```diff
@@ -161,11 +170,17 @@ class Task : public QObject, public QRunnable {
     }
 
    protected:
+    /*!
+     * The task subclass must implement this method. This method is called to start to run the task.
+     * The task is not finished when this method returns. the subclass must manually call emitSucceeded() or emitFailed() instead.
+     */
     virtual void executeTask() = 0;
 
    protected slots:
+    //! The Task subclass must call this method when the task has succeeded 
     virtual void emitSucceeded();
     virtual void emitAborted();
```

Comment:
//! The Task subclass must call this method when the task was aborted

Reply:
abort also checks the canAbort() function and should always be called in external code.
this function should be called by the internal task code